ext/drx/CMakeLists.txt

```diff
@@ -62,6 +62,9 @@ configure_extension(drx OFF)
 use_DynamoRIO_extension(drx drcontainers)
 use_DynamoRIO_extension(drx drmgr)
 use_DynamoRIO_extension(drx drreg)
+if (AARCH64)
+    use_DynamoRIO_extension(drx drutil)
```

Comment:
Adding a drutil dependence to drx is problematic for the licenses: drx was created to be separate from drutil's LGPL license (which was there because it came from Dr. Memory which is LGPL).  So this dependence kind of defeats the purpose of that license split.  Most of our large tools end up using drutil anyway, but this may affect some external user.

Is there an alternative?

This should at least be well-documented: these places would have to be updated:
```
ext/drx/README.md:utilities for instrumentation and sports a BSD license, as opposed to the drutil Extension
ext/drx/drx.dox:instrumentation and sports a BSD license, as opposed to the \p drutil Extension
```

Reply:
Thanks. I didn't realise. I will have a look for an alternative solution.

Reply:
Fixed by PR: #6389